util/proposal.go

```diff
@@ -36,3 +37,22 @@ func GetProposalName(payload string) (string, error) {
 func IsValidProposalName(str string) bool {
 	return validProposalName.MatchString(str)
 }
+
+// MountValidRegExpProposalName return a regex string for matching the proposal name
```

Comment:
I think we should rename this to like: CreateProposalTitleRegex.  And it needs to be "returns" instead of "return".